### PR TITLE
set deno-dev base  ubuntu image to focal

### LINKED ISSUE #75: Upgrade deno image to ubuntu 20.04
Build is currently failing: https://jenkins.jonathan-boudreau.com/blue/organizations/jenkins/docker-dev/detail/docker-dev/34/pipeline/

### PR DESCRIPTION
**Pull request by hardy613**
fixes #75 -  update build.py with deno-dev base image to focal

### CODE REVIEW

File: build.py [hunk 1]
```diff
@@ -43,7 +44,7 @@
     {'name': 'pg-dev', 'tag': '11', 'args': {'PG_VERSION': '11'}},
     {'name': 'pg-dev', 'tag': '12', 'args': {'PG_VERSION': '12'}},
     {'name': 'mongo-dev', 'tag': '4.1', 'args': {'BASE_TAG': '4.1'}},
-    {'name': 'redis-dev', 'tag': '6'}
+    {'name': 'redis-dev', 'tag': '6', 'path': 'redis'}
```
Comment (AGhost-7) [inline build.py:47]: I moved the folder in the correct location instead. It doesn't need the path specified anymore.

**Review by AGhost-7 — CHANGES_REQUESTED** [review]
Are you sure this works? Auto-complete seems broken among other things.

**Comment by hardy613** [comment]
> Are you sure this works? Auto-complete seems broken among other things.

No. should have been set to draft. Apologies. 

Autocomplete might be related to recent deno updates - might need to generate deno types. Also, deno is in rapid development and new releases happen often, what would be the best way to manage a fast-paced environment like deno?

**Comment by AGhost-7** [comment]
> Autocomplete might be related to recent deno updates - might need to generate deno types
I get a YCM error when I try to run `:YcmCompleter GoTo` in neovim.

> Also, deno is in rapid development and new releases happen often, what would be the best way to manage a fast-paced environment like deno?
For Rust I just used different channels (one for nightly and one for stable). Is there something similar for deno?

**Comment by hardy613** [comment]
YCM seems to have a hard time installing still and the error gets mangled by installing the NVIM Plugs

**Comment by AGhost-7** [comment]
@hardy613 Try installing manually in that case. You can start a container up using your base image (`aghost7/nvim:focal`) and run your script line by line.

**Comment by hardy613** [comment]
`:YcmCompleter GoTo` working

Screenshot from 2020-12-20 19-46-14

with nice property hints

Screenshot from 2020-12-20 19-45-23

**Review by AGhost-7 — APPROVED** [review]

**Pull request merged**